scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
We usually want to avoid as many problems as possible, so let's go by default with 

```suggestion
set -euo pipefail
```

and disable specific options only when necessary.

Reply:
Done